src/lib/hyrise.hpp

```diff
@@ -38,6 +38,8 @@ class Hyrise : public Singleton<Hyrise> {
   MetaTableManager meta_table_manager;
   SettingsManager settings_manager;
   Topology topology;
+
+  // We need to pass the SettingsManager here to allow the correct registration of LogManager's setting.
```

Comment:
Rather: We need to explicitly pass `settings_manager` as the Hyrise object is not fully initialized yet.